### DISCUSSION TITLE
Welcome to radar-prmt-android Discussions!

### DISCUSSION THREAD

**Discussion by afolarin** [RADAR-CNS/radar-prmt-android#121] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shashankRanaNW** [comment]
I wanted to know whether radar-base, and especially the android part would be participating in the gsoc 23 program? I tried to find the developer community of radar base to ask about this, but I wasn't able to find it, also the slack link in gsoc website was broken.

Also can you tell about the community of Radar base devs?
I wasn't able to find one on the net.

Thanks

**Comment by afolarin** [comment]
Hi,
We are planning to participate in GSoC 2023, I believe we already have a few projects drafted. Normally, we'll post on the RADAR-base Slack channel https://radar-base.org/ you can find the signup link on the Docs menu.

**Reply by shashankRanaNW** [reply]
Thanks man, got into the slack channel
btw, you guys prefer github discussions or slack?
RADAR-Kubernote's git discussions seems pretty active, not so in here.

**Comment by afolarin** [comment]
Yes, this is a Slack vs Github Discussions something of a debate. The Slack channel predates the availability of Discussions on GitHub and in many ways is better, but, Slack has a momentum of legacy behind it and also has a bunch of informal discussion channels which people use for various functions. Likely we will keep both running but we _are_ generally trying to get discussions moved onto Github for the components of RADAR-base. So feel free for now to use both you can also link your discussions in the Slack channel if you want to draw attention to it.
